kostyor/db/api.py

```diff
@@ -120,6 +120,13 @@ def create_host(name, cluster_id):
             'cluster_id': new_host.cluster_id}
 
 
+def get_hosts_by_cluster(cluster_id):
+    hosts = db_session.query(models.Host).filter_by(
+        cluster_id=cluster_id).all()
+    return {'hosts': list(map(lambda x: {'id': x.id, 'name': x.hostname},
```

Comment:
Hm.. In my opinion it would be better to do not introduce intermediate `hosts` key and return a list of hosts directly. Also, please use `to_dict()` on the database object instead of `list + map + lamda` slice.